### PR TITLE
Move manual backup docs

### LINKED ISSUE #118: [DOC] How to create a manual backup
### Preliminary Checks

- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/nebari-dev/nebari/discussions

### Summary

[notion doc](https://www.notion.so/quansightlabs/Create-manual-backup-bbec33ec945c4b1f9bafae9627d40ef3)

Links to old docs:
https://docs.qhub.dev/en/latest/source/admin_guide/backup.html
https://docs.qhub.dev/en/latest/source/admin_guide/breaking-upgrade.html 

Related to Upgrading Nebari (#17)

### Steps to Resolve this Issue

.

### PR DESCRIPTION
**Pull request by kcpevey**
## Reference Issues or PRs

Resolves #118 

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features to not work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [ ] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [ ] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

**Comment by viniciusdc** [comment]
@kcpevey, the preview is failing, do you know why that might be happening?

File: docs/docs/how-tos/manual-backup.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+
+There is no need to TAR anything up since the only file required to be backed up is `/srv/jupyterhub/jupyterhub.sqlite`.
+
+### Backing up JupyterHub DB
+
+Now we just need to upload the file to S3. You might want to [install the AWS CLI tool](#installations) as we did before, however, as the Hub container is a rather restricted
+environment the recommended approach is to upload files to AWS S3 buckets using curl.
+
+For more details please refer to the [using curl to access AWS S3 buckets](./awss3curl.md) documentation.
+
```
Comment (viniciusdc) [inline docs/docs/how-tos/manual-backup.md:232]:
```suggestion
For more details please refer to the [using curl to access AWS S3 buckets] documentation.

```

File: docs/docs/how-tos/manual-backup.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+You will need to overwrite the file `/srv/jupyterhub/jupyterhub.sqlite` based on the version backed up to S3.
+
+You should restart the pod:
+
+```bash
+kubectl delete -n dev pod hub-765c9488d6-8z4nj
+```
+
+As for uploads, [you may need to use curl to download items from an AWS S3 bucket](./awss3curl.md).
+
```
Comment (viniciusdc) [inline docs/docs/how-tos/manual-backup.md:244]:
```suggestion
As for uploads, [you may need to use curl to download items from an AWS S3 bucket]

```

File: docs/docs/how-tos/manual-backup.md [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+Nebari provides a simple script to export the important user/group database. Your new Nebari cluster will recreate a lot of Keycloak config (including new Keycloak clients which will
+have new secrets), so only the high-level Group and User info is exported.
+
+If you have a heavily customized Keycloak configuration, some details may be omitted in this export.
+
+### Export Keycloak
+
+The export script is at [`nebari/scripts/keycloak-export.py`](https://github.com/Quansight/qhub/blob/main/scripts/keycloak-export.py).
+
```
Comment (viniciusdc) [inline docs/docs/how-tos/manual-backup.md:255]:
```suggestion
The export script is at [`nebari/scripts/keycloak-export.py`](https://github.com/nebari-dev/nebari/blob/main/scripts/keycloak-export.py).

```

File: docs/docs/how-tos/manual-backup.md [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+```shell
+python nebari/scripts/keycloak-export.py -c nebari-config.yaml > exported-keycloak.json
+```
+
+You may wish to upload the Keycloak export to the same S3 location where you uploaded the TAR file in the NFS section.
+
+### Import Keycloak
+
+To re-import your users and groups, [login to the /auth/ URL](../installation/login.md) using the root username and password.
+
```
Comment (viniciusdc) [inline docs/docs/how-tos/manual-backup.md:273]:
```suggestion
To re-import your users and groups, [login to the /auth/ URL] using the root username and password.

```

**Review by viniciusdc — COMMENTED** [review]
Removing links that do not exists for now, will add replacements later

**Comment by kcpevey** [comment]
Ah, yes. I intended to take care of the CI failures for you after you added your text edits, but it seems you did that first :) I'm happy to take care of any formatting or anything else. I just need help with content.

**Comment by viniciusdc** [comment]
@kcpevey, the above docs are up to date and will work with the migration. Though I can foresee we will need to add more to this later, once we migrate beta to nebari-v2022.10.1 we will be able to add more to this. For now, to get thing unblocked for the release this is good enough :smile:

**Review by viniciusdc — APPROVED** [review]

**Pull request merged**